### ISSUE TITLE
Getting `warning: class variable access from toplevel` with bundler

### ISSUE THREAD

**Issue by clowder** [rodjek/puppet-lint#85]
Pretty simply if I run `bundle exec rake lint` I get the output below, if I just do a `rake lint` then puppet-lint runs fine. Is there a config step I’m missing? Or a bug?

``` bash
$ bundle exec rake lint --trace
~/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/gems/puppet-2.6.12/lib/puppet/util/log/destinations.rb:99: warning: class variable access from toplevel
rake aborted!
~/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/gems/puppet-2.6.12/lib/puppet/application/kick.rb:105: Invalid next
~/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/gems/puppet-2.6.12/lib/puppet/application/kick.rb
```

---

**Gemfile**

``` ruby
source :rubygems

gem 'rake'
gem 'puppet', '2.6.12'
gem 'puppet-lint'
```

**Rakefile**

``` ruby
require 'puppet-lint/tasks/puppet-lint'
```
